packages/hydrogen-react/src/storefront-api-response.types.ts

```diff
@@ -15,7 +15,6 @@ type StorefrontApiExtensions = {
     | 'ACCESS_DENIED'
     | 'SHOP_INACTIVE'
     | 'INTERNAL_SERVER_ERROR'
-    // eslint-disable-next-line @typescript-eslint/ban-types -- This enables autocomplete for the defined error codes we have, but also allows for any string to be used, too. https://github.com/ascorbic/unpic-img/blob/c5de1bf01abd85ef7e8ac07c40c19128a3097e69/packages/core/src/core.ts#L40 Thanks Matt Kane!
```

Comment:
This is a generated file that we commit into the package. If the lint is complaining about this file, I think it's better to put it in eslint ignore list.

When we download the schema, we output with pretty print so that it's read-able

Reply:
yes, eslint was complaining since we don't use the `@typescript-eslint` plugin for `hydrogen-react` (but there is a directive in this file that refers to this plugin). I added the file to the ignore list